docs/source/tutorials/code_samples.rst

```diff
@@ -59,4 +59,8 @@ Code samples are used for automated testing. They are also a great starting poin
 - :ref:`System information` - Displays device system information (memory/cpu usage, temperature)
 - :ref:`OpenCV support` - Demonstrates how to retrieve an image frame as an OpenCV frame
 - :ref:`Device Queue Event` - Demonstrates how to use device queue events
-- :ref:`Queue add callback` - Demonstrates how to use queue callbacks
\ No newline at end of file
+- :ref:`Queue add callback` - Demonstrates how to use queue callbacks
+- :ref:`Calibration Flash v5` - Demonstrates how to write API over XLink (CalibrationHandler is v5)
```

Comment:
> Demonstrates how to write API over XLink (CalibrationHandler is v5)
This should be "Demonstrates how to flash Calibration data of Version 5 (.calib file from gen1) to the device.

Reply:
This doesn't demonstrate how to write the API. it demonstrates how to use it. Any specific reason why these comments were resolved @csaba-luxonis  ?

Reply:
Fixed it in mixed samples but not in that one, thanks for noticing it